e2e/tests/messageJudge_test.js

```diff
@@ -28,6 +29,7 @@ Scenario('HMCTS admin messages the judge', async ({I, caseViewPage, messageJudge
   I.seeInTab(['Message 1', 'Requested by'], 'Swansea city council');
   I.seeInTab(['Message 1', 'Message'], 'Some note');
   I.seeInTab(['Message 1', 'Status'], 'Open');
+  I.dontSee('Closed messages');
```

Comment:
minor: maybe that's a slower compare to a selector

Reply:
updated the test to use a selector